gpii/node_modules/matchMakerFramework/src/MatchMakerFramework.js

```diff
@@ -44,7 +42,7 @@
         invokers: {
             preProcess: {
                 funcName: "gpii.matchMakerFramework.preProcess",
-                args: ["{that}", "{arguments}.0"]
+                args: ["{that}", "{flowManager}", "{arguments}.0"]
```

Comment:
We can't do this. Having the entire flowManager as a dependency of the matchMakerFramework is faulty